broadinstitute/single_cell_portal_core
# Missed some cleanup of reference images

Missed some files around reference images which is why staging is currently broken
This finds and removes those remaining pieces

Utilized: https://github.com/broadinstitute/single_cell_portal_core/commit/f8b4cdc3bc8e9372947d01a8333c90ac2c151f79#diff-ff4757fbd05349b697a855bde79438e30252a0df7191de9205c1a4d7d46a16a9 for context to find the rest

To test
- pull this branch and go to any study explore page
- confirm page loads no errors
